tests/integrations/rediscluster/test_rediscluster.py

```diff
@@ -0,0 +1,51 @@
+from sentry_sdk import capture_message
+import sentry_sdk.integrations
+from sentry_sdk.integrations.redis import RedisIntegration
+
+import rediscluster
+
+
+def _test_rediscluster_basic(sentry_init, capture_events, monkeypatch, clsname):
```

Comment:
Please instead of using this helper function use `pytest.mark.parametrize`.

Reply:
How would that look like with the test if the `StrictRedisCluster` class is present?
https://github.com/getsentry/sentry-python/pull/752/files#diff-52fd8f5367c660c1f800f2cb58148e7eR49-R51

Reply:
Figured something out.